aws/dynamodb.go

```diff
@@ -0,0 +1,71 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func getAllDynamoTables(session *session.Session) ([]*string, error) {
+	var TableNames []*string
```

Comment:
Please use lower case for private variables: e.g., `tableNames`. Go uses upper case for public variables.